jobrunner/cli/flags.py

```diff
@@ -0,0 +1,80 @@
+"""
+Ops utility getting and setting db flags
+"""
+import argparse
+import sqlite3
+import sys
+from dataclasses import dataclass
+
+from jobrunner.lib.database import create_table, get_connection, get_flag, set_flag
+
+
+NOVALUE = object()
+
+
+@dataclass
+class Flag:
+    """Do you have a flaaaaaag?"""
+
+    name: str
+    value: str = None
+
+    @classmethod
+    def from_cli(cls, raw):
+        name, equals, value = raw.partition("=")
+        if value == "":
+            if equals == "=":
+                # flag=, set flag to None
+                value = None
+            else:
+                value = NOVALUE
+        return cls(name, value)
+
+
+def main(flags, create):
+    try:
+        do_flags(flags)
+    except sqlite3.OperationalError as e:
+        if "no such table" in str(e):
+            if create:
+                create_table(get_connection(), "flags")
+                do_flags(flags)
+            else:
+                sys.exit(
+                    "Flags table does not exists. Run command again with --create to create it."
+                )
+        else:
+            raise
+
+
+def do_flags(flags):
+    if not flags:
+        all_flags = get_connection().execute("SELECT name FROM flags")
+        flags = [Flag(f[0], NOVALUE) for f in all_flags]
+    for flag in flags:
+        if flag.value is not NOVALUE:
+            set_flag(flag.name, flag.value)
+        print(f"{flag.name}={get_flag(flag.name)}")
+
+
+def run(argv):
+    parser = argparse.ArgumentParser(description=__doc__.partition("\n\n")[0])
+    parser.add_argument(
+        "flags",
+        metavar="FLAG[=VALUE]",
+        nargs="*",
+        type=Flag.from_cli,
+        help="Just flag to view, or flag=value to set, flag= to clear",
```

Comment:
This API makes me a bit nervous. I'd prefer explicit `flags get` and `flags set` commands.

Reply:
How do you imagine it going wrong?

Reply:
Changed. I'm not entirely happy with it, but it'll do for now